### PR TITLE
New prettier game menu with game settings button

### PR DESCRIPTION
**Pull request by PestoVerde322**
## About The Pull Request

This PR, aside making the menu prettier overall, adds a game settings tab as, currently, it's not very well seen unless you know where to look for.

Inspired off by Daedalus station's current game menu.

## Why It's Good For The Game

Fancier menu and also, easy to access game settings.

## Testing Photographs and Procedure
<details>
<summary>Screenshots&Videos</summary>

image

</details>

## Changelog
:cl:
add: Added a button to access the game settings from the menu
code: Made overall the game menu prettier🐝
/:cl:

### CODE REVIEW

**Review by tontyGH — APPROVED** [review]
LGTM (I didn't read any of the code)

**Comment by itsmeow** [comment]
the "prettyness" in question: two <hr /> 💀

**Comment by tontyGH** [comment]
It does look a little nicer though

**Comment by Kapu1178** [comment]
If you're gonna 1:1 rip our menu atleast credit us

**Comment by itsmeow** [comment]
the stolen <hr /> tags:

**Comment by Crossedfall** [comment]
> If you're gonna 1:1 rip our menu atleast credit us

🤡 📯

File: code/world.dm [hunk 1]
```diff
@@ -5,7 +5,7 @@
 	mob = /mob/dead/new_player
 	turf = /turf/open/space/basic
 	area = /area/space
-	view = "17x15"
+	view = "15x15"
```
Comment (Crossedfall) [inline code/world.dm]: Why is this changing?

File: config/config.txt [hunk 2]
```diff
@@ -530,9 +530,9 @@ DISABLE_HIGH_POP_MC_MODE_AMOUNT 100
 ## Uncomment to set the number of /world/Reboot()s before the DreamDaemon restarts itself. 0 means restart every round. Requires tgstation server tools.
 ROUNDS_UNTIL_HARD_RESTART 3
 
-##Default screen resolution, in tiles.
+##Default view size, in tiles.
 ##	By default, this is 15x15, which gets simplified to 7 by BYOND, as it is a 1:1 screen ratio.
-##	For reference, Goonstation uses a resolution of 21x15 for it's widescreen mode.
+##  15x15 would be the standard square view. 21x15 is what goonstation uses for widescreen.
```
Comment (Crossedfall) [inline config/config.txt]: This also seems like an irrelevant change

File: code/modules/mob/dead/new_player/new_player.dm [hunk 3]
```diff
@@ -384,7 +392,7 @@
 
 
 /*
-	Ported from yogs: https://github.com/yogstation13/Yogstation-TG/blob/master/yogstation/code/modules/mob/dead/new_player/new_player.dm
+	Ported from yogs: https://github.com/yogstation13/Yogstation-TG/blob/master/yogstation/code/modules/mob/dead/new_player/new_player.dm //this link is dead since yog no longer has a new_player.dm!
```
Comment (Crossedfall) [inline code/modules/mob/dead/new_player/new_player.dm]: This also feels entirely unrelated, but if you're going to update it just use the permanent link to the old file.

```suggestion
	Ported from yogs: https://github.com/yogstation13/Yogstation/blob/5bffae65b7421766c7397c0e457989eaf7c92f5d/yogstation/code/modules/mob/dead/new_player/new_player.dm
```

**Review by Crossedfall — CHANGES_REQUESTED** [review]
The handful of irrelevant changes should be removed.

File: code/world.dm (hunk 1 above)
Comment (PestoVerde322) [reply code/world.dm]: Leftover Accident

File: config/config.txt (hunk 2 above)
Comment (PestoVerde322) [reply config/config.txt]: Leftover Accident

**Review by PowerfulBacon — CHANGES_REQUESTED** [review]
The bee needs to be big enough to look like a bee, or needs to be removed. Right now it looks like a manhack from HL2

**Comment by francinum** [comment]
> The bee needs to be big enough to look like a bee, or needs to be removed. Right now it looks like a manhack from HL2

I thought it looked like a wheelchair.
